.gitignore

```diff
@@ -1,10 +1,8 @@
-# Transpiled CSS files
-
+# Showcase directory
 /showcase
```

Comment:
Why showcase is in gitignore? :thinking:

Reply:
I tried to come up with some showcase structure, but haven't really put anything in there yet, so I decided to place it in gitignore at the moment. Should I push it anyway?

Reply:
That'a antipattern. If you want to add unfinished showcase that's ok, but if you don't then just remove it from gitignore and also from the files.